mappings/net/minecraft/world/GameMode.mapping

```diff
@@ -21,4 +21,6 @@ CLASS none/ait net/minecraft/world/GameMode
 	METHOD a setAbilitites (Lnone/aaj;)V
 		ARG 0 abilities
 	METHOD b getName ()Ljava/lang/String;
+	METHOD c isAdventureOrSpectator ()Z
```

Comment:
That might get confusing. Also, it would break the format.

My policy is to not map names unless I'm sure... and you can always poke me to look at something if you need it.